app/workers/import_worker.rb

```diff
@@ -0,0 +1,15 @@
+class ImportWorker
+  include Sidekiq::Worker
+
+  def perform(device, json_file)
```

Comment:
It was much faster when looping over a csv file so i'm pretty sure it's the array related stuff (each, split, attr[1] etc.)